lib/courseUtil.js

```diff
@@ -0,0 +1,73 @@
+const ERR = require('async-stacktrace');
+const { sqlDb, sqlLoader } = require('@prairielearn/prairielib');
+const child_process = require('child_process');
+
+const config = require('./config');
+
+const sql = sqlLoader.loadSqlEquiv(__filename);
+
+function getCommitHash(course, callback) {
+    const gitEnv = process.env;
+    if (config.gitSshCommand != null) {
+        gitEnv.GIT_SSH_COMMAND = config.gitSshCommand;
+    }
```

Comment:
Not that it hurts, but I think we don't care about GIT_SSH_COMMAND here because this will be a purely local operation.

Reply:
Gotcha - I wasn't completely sure what the purpose of that was, I was just copying from the other places that git operations are done.

Reply:
It's for the containerization code. In current production we are running git as the root user and we've configured the root ssh keys to allow access to github. In the containerized versions we are mounting these keys into the the container and so we need to pass a git command line argument to point at the keys. This avoids hard-coding root ssh keys into our cloudformation scripts.